src/hypersync/IDataSource.ts

```diff
@@ -39,6 +40,7 @@ export interface IDataSource {
   getData<TData = DataObject>(
     dataSetName: string,
     params?: DataValueMap,
+    page?: string,
```

Comment:
Needs a param tag.

Also, this optional param is only used for IDataSetResultComplete and the metadata business is only used for IDataSetResultPending, yeah?  And it will always be that way?

If so, might include that in the param tag comments.